build/system.gyp

```diff
@@ -35,6 +35,24 @@
        ],
       ],
     },
+    {
+      'target_name': 'libnotify',
+      'type': 'none',
+      'conditions': [
+        ['building_crosswalk_bin==1', {
+          'defines': ['USE_LIBNOTIFY=1'],
+          'cflags': [
+            '<!@(pkg-config --cflags libnotify)',
+          ],
+          'link_settings': {
+	    'libraries': [
```

Comment:
Same thing about following the "gio" target: splitting `libraries` into `ldflags` and `libraries` is a good idea (plus your indentation here is wrong, you used 1 space instead of 2).